platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gossip/SyncGossip.java

```diff
@@ -188,9 +188,10 @@ protected SyncGossip(
         final ThreadConfig threadConfig = platformContext.getConfiguration().getConfigData(ThreadConfig.class);
 
         final BasicConfig basicConfig = platformContext.getConfiguration().getConfigData(BasicConfig.class);
-
-        topology = new StaticTopology(random, addressBook, selfId, basicConfig.numConnections());
         final List<PeerInfo> peers = Utilities.createPeerInfoList(addressBook, selfId);
+
+        topology =
+                new StaticTopology(random, peers, addressBook.getIndexOfNodeId(selfId), basicConfig.numConnections());
```

Comment:
this index is an AB concept, so we should get rid of it

Reply:
@lpetrovic05 Is the scope of this PR to simply remove the address book references or also drop our support for non-fully connected network topologies?   The `RandomGraph` class uses hard-coded arrays of size `addressbook.getSize()`.  The translation from `id` to `index` is to support the use of `RandomGraph` to decide who is a neighbor when there are more nodes in the network than the degree of edges allowed per node.  

If we drop the use of `RandomGraph`, then we can get rid of the translation from `id` to `index`.  But if we keep `RandomGraph`, then we still need to translate from `id` to `index`.  We can do it without the address book, using the `List<PeerInfo>`, but we'll have to reconstruct the `Map<id, index>`, and be sensitive to where the `selfId` falls into the sequence.

Reply:
`RandomGraph` has now been dropped with the network becoming fully connected. Resolving.